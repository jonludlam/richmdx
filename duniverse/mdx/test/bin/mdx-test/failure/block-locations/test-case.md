There was a bug in a the lexer/parser for markdown and cram files
that lead to increasingly big offsets in block locations.
This impacted error reporting quite badly.

Here we have a few valid blocks followed by one that triggers an error,
it should be reported with the correct block location.


<!-- $MDX toplevel -->
```ocaml
# 1 + 1;;
```

<!-- $MDX toplevel -->
```ocaml
# 1 + 1;;
```

<!-- $MDX toplevel -->
```ocaml
# 1 + 1;;
```

<!-- $MDX pif=paf -->
```ocaml
# 1 + 1;;
```
