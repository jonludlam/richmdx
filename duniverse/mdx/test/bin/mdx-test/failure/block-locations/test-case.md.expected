[mdx] Fatal error: File "test-case.md", lines 24-27: invalid code block: `pif` is not a valid label.
